src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html,
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  height: 40px;
+  margin: auto 0;
```

Comment:
Ok, but as for me it's more logical to set `align-items` to parent instead (usually aligning by cross axis applied to all elements)